go/base/decorators.py

```diff
@@ -0,0 +1,24 @@
+"""Decorator utilities."""
```

Comment:
Hmm, if its unused I vote we remove it, since we'll always have it in our git history if we find we would like to use it.